setup.py

```diff
@@ -19,7 +19,7 @@
     os.system("git push --tags")
     sys.exit()
 
-with io.open('README.rst', 'r', encoding='utf-8') as readme_file:
+with io.open('README.md', 'r', encoding='utf-8') as readme_file:
```

Comment:
⚠️ Since Readme has been converted to markdown, it is also necessary to inform setuptools which markdown is being used. Just add `long_description_content_type='text/markdown'` to the `setup` (after line 47)

Read more here: https://packaging.python.org/guides/making-a-pypi-friendly-readme/

Reply:
Fixed on e986419